src/client/sound_openal.cpp

```diff
@@ -379,6 +377,23 @@ class OpenALSoundManager: public ISoundManager
 		infostream << "Audio: Deinitialized." << std::endl;
 	}
 
+	u16 getFreeId() const
+	{
+		static thread_local u16 last_used_id = 0;
```

Comment:
why a static thread_local? This feels more like a class member thing

Reply:
Recycled 1:1 from activeobjectmgr.h

Reply:
Hmm,  I think using class members is less of a code small, but I'm not too bothered about it.

Reply:
Updated.